src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssPropertiesAndValues = sourceString.split(';');
+  const result = {};
+
+  cssPropertiesAndValues.forEach((propertyValuePair) => {
+    const trimmedItem = propertyValuePair.trim();
+
+    if (trimmedItem.length < 4) {
+      return;
+    }
+
+    const [key, value] = trimmedItem.split(':');
```

Comment:
When splitting the 'trimmedItem' by ':', consider using a regular expression that includes whitespace around the colon to ensure that any extra spaces are not included in the key or value, which would make the trimming on line 22 redundant.